MaloWBotLK/MaloWBotLK.lua

```diff
@@ -197,6 +197,15 @@ function mb_onUpdate()
 		return
 	end
 	if mb_commanderUnit ~= nil and mb_shouldFollow then
+		if CheckInteractDistance(mb_commanderUnit, 2) == nil then
+			FollowUnit(mb_commanderUnit)
+			return
+		end
+
+		if UnitBuff("player", "Drink") then
+			return
+		end
+
```

Comment:
I would move all these changes to be inside mb_Drink(), and inside that modify the mb_shouldFollow variable to true/false to get the characters to temporarily stop follow while drinking.